roles/cleanup_gs_data/templates/cleanup_gs.bash

```diff
@@ -0,0 +1,39 @@
+# If folder is empty for 1 week -> mail helpdesk
+# If folder is empty for 2 weeks -> delete
+# If folder is not empty, but no .finished file is present (1 week) -> mail helpdesk
+# If folder is not empty, but no .finished file is present (2 weeks) -> delete
```

Comment:
Do we need notifications for empty folders? If they are empty, that suggests the data was processed correctly leaving just that empty folder behind. I would simply delete empty folders already after 2 days instead of after 2 weeks without sending any notifications.

Reply:
I'm not sure. Could be that the folder is already there, but the data is not?